### PR TITLE
Update classifier-tc-definitions.md

### PR DESCRIPTION
**Pull request by jenli311**
Content updates:
- Added content on regulatory compliance classifiers going to public preview
- Added note on regulatory compliance classifier behavior observed during private preview at the end of classifier list

### CODE REVIEW

**Comment by jenli311** [comment]
@chrfox here is the PR for updating trainable classifiers page with new regulatory compliance classifiers

File: microsoft-365/compliance/classifier-tc-definitions.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Microsoft 365 comes with multiple pre-trained classifiers:
 
 - **Construction specifications (preview)**: Detects construction specifications for commercial and industrial projects like factories, plants, commercial offices, airports, roads. Captures guidelines on the quality, quantity, types of building material, processes etc. Detects content in .docx, .docm, .doc, .dotx, .dotm, .dot, .pdf, .rtf, .txt, .one, .msg, .eml, .pptx, .pptm, .ppt, .potx, .potm, .pot, .ppsx, .ppsm, .pps, .ppam, .ppa files.
 
+- **Corporate Sabotage (preview)**: Detects messages that may mention acts to damage or destroy  corporate assets or property. This classifier can help customers manage regulatory compliance obligations such as NERC Critical Infrastructure Protection standards or state by state regulations like Chapter 9.05 RCW in Washington state. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-tc-definitions.md]:
```suggestion
- **Corporate Sabotage (preview)**: Detects messages that may mention acts to damage or destroy corporate assets or property. This classifier can help customers manage regulatory compliance obligations such as NERC Critical Infrastructure Protection standards or state-by-state regulations like Chapter 9.05 RCW in Washington state. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files. 
```

File: microsoft-365/compliance/classifier-tc-definitions.md [hunk 2]
```diff
@@ -54,6 +56,8 @@ Microsoft 365 comes with multiple pre-trained classifiers:
 
 - **Financial Statement (preview)**: Detects financial statements like income statement, balance sheet, cash flow statement, statement of changes in equity. Detects content in .docx, .docm, .doc, .dotx, .dotm, .dot, .pdf, .rtf, .txt, .xlsx, .xlsm, .xlsb, .xls, .csv, .xltx, .xltm, .xlt, .xlam, .xla files.
 
+- **Gifts & entertainment (preview)**: Detects messages that may suggest exchanging gifts or entertainment in return for service, which violates regulations related to bribery. This classifier can help customers manage regulatory compliance obligations such as  Foreign Corrupt Practices Act (FCPA), UK Bribery Act and FINRA Rule 2320. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-tc-definitions.md:62]:
```suggestion
- **Gifts & entertainment (preview)**: Detects messages that may suggest exchanging gifts or entertainment in return for service, which violates regulations related to bribery. This classifier can help customers manage regulatory compliance obligations such as Foreign Corrupt Practices Act (FCPA), UK Bribery Act, and FINRA Rule 2320. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```

File: microsoft-365/compliance/classifier-tc-definitions.md [hunk 3]
```diff
@@ -80,6 +84,7 @@ Microsoft 365 comes with multiple pre-trained classifiers:
 
 - **Meeting Notes** (preview): This classifier detects meeting notes. Detects content in .docx, .docm, .doc, .dotx, .dotm, .dot, .pdf, .rtf, .txt, .one, .msg, .eml, .pptx, .pptm, .ppt, .potx, .potm, .pot, .ppsx, .ppsm, .pps, .ppam, .ppa files.
 
+- **Money laundering (preview)**: Detects signs that may suggest money laundering or engagement in acts to conceal or disguise the origin or destination of proceeds. This classifier can help customers manage regulatory compliance obligations such as the Bank Secrecy Act, the USA Patriot Act, FINRA Rule 3310 and Anti-Money Laundering Act of 2020. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-tc-definitions.md:92]:
```suggestion
- **Money laundering (preview)**: Detects signs that may suggest money laundering or engagement in acts to conceal or disguise the origin or destination of proceeds. This classifier can help customers manage regulatory compliance obligations such as the Bank Secrecy Act, the USA Patriot Act, FINRA Rule 3310, and the Anti-Money Laundering Act of 2020. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```

File: microsoft-365/compliance/classifier-tc-definitions.md [hunk 4]
```diff
@@ -93,11 +98,12 @@ Microsoft 365 comes with multiple pre-trained classifiers:
 
 - **Profanity**: Detects a specific category of offensive language text items that contain expressions that embarrass most people. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
 
+- **Regulatory collusion (preview)**: Detects    messages that may violate regulatory anti-collusion requirements such as an attempted concealment of sensitive information. This classifier can help customers manage  regulatory compliance obligations such as the Sherman Antitrust Act, Securities Exchange Act 1933  , Securities Exchange Act of 1934, Investment Advisers Act of 1940, Federal Commission Act, and Robinson-Patman Act. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-tc-definitions.md]:
```suggestion
- **Regulatory collusion (preview)**: Detects messages that may violate regulatory anti-collusion requirements such as an attempted concealment of sensitive information. This classifier can help customers manage regulatory compliance obligations such as the Sherman Antitrust Act, Securities Exchange Act 1933, Securities Exchange Act of 1934, Investment Advisers Act of 1940, Federal Commission Act, and Robinson-Patman Act. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.  
```

File: microsoft-365/compliance/classifier-tc-definitions.md [hunk 5]
```diff
@@ -107,18 +113,27 @@ Microsoft 365 comes with multiple pre-trained classifiers:
 
 - **Statement of Work (preview)**: This classifier detects statement of work containing details like requirements, responsibilities, terms and conditions for both parties. Detects content in .docx, .docm, .doc, .dotx, .dotm, .dot, .pdf, .rtf, .txt files.
 
+- **Stock manipulation (preview)**: Detects signs of possible stock manipulation, such as recommendations to buy, sell or hold stocks that may suggest an attempt to manipulate the stock price. This classifier can help customers manage regulatory compliance obligations such as the Securities Exchange Act of 1934, FINRA Rule 2372, and FINRA Rule 5270. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-tc-definitions.md]:
```suggestion
- **Stock manipulation (preview)**: Detects signs of possible stock manipulation, such as recommendations to buy, sell, or hold stocks that may suggest an attempt to manipulate the stock price. This classifier can help customers manage regulatory compliance obligations such as the Securities Exchange Act of 1934, FINRA Rule 2372, and FINRA Rule 5270. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.  
```

File: microsoft-365/compliance/classifier-tc-definitions.md [hunk 6]
```diff
@@ -107,18 +113,27 @@ Microsoft 365 comes with multiple pre-trained classifiers:
...
 
 - **Threat**: Detects a specific category of offensive language text items related to threats to commit violence or do physical harm or damage to a person or property. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
 
+- **Unauthorized disclosure (preview)**: Detects sharing of information containing content that is explicitly designated as confidential or internal to unauthorized individuals. This classifier can help customers manage regulatory compliance obligations such as FINRA Rule 2010 and SEC Rule 10b-5. Detects content in .msg, .docx, .pdf, .txt, .rtf, .jpeg, .jpg, .png, .gif, .bmp, .svg files.
+
+> [!NOTE]
+> The **Corporate sabotage, Gifts & entertainment, Money laundering, Regulatory collusion, Stock Manipulation, and Unauthorized disclosure classifiers** may capture a large volume of bulk sender/newsletter content due to a known issue in development. This behavior will be resolved in January 2023. 
+
+> For a temporary solution to address large volumes of bulk sender/newsletter content, add the **Message is not sent to any of these domains condition** with a list of domains to exclude.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-tc-definitions.md]:
```suggestion
> The **Corporate sabotage, Gifts & entertainment, Money laundering, Regulatory collusion, Stock Manipulation, and Unauthorized disclosure classifiers** may capture a large volume of bulk sender/newsletter content due to a known issue in development. This behavior will be resolved in the near future. 
>
> For a temporary solution to address large volumes of bulk sender/newsletter content, add the **Message is not sent to any of these domains condition** with a list of domains to exclude. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jenli311, kindly commit the suggested changes. Please note that Microsoft docs does not publish definite dates as dates and plans may slip. Many thanks

**Review by chrfox — COMMENTED** [review]
@jenli311 please review and approve these changes. When do you want these to go live?

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/compliance/classifier-tc-definitions.md (hunk 2 above)
Comment (jenli311) [reply microsoft-365/compliance/classifier-tc-definitions.md:62]: good catch!

File: microsoft-365/compliance/classifier-tc-definitions.md (hunk 3 above)
Comment (jenli311) [reply microsoft-365/compliance/classifier-tc-definitions.md:92]: good catch, agree with changes

**Review by jenli311 — COMMENTED** [review]
changes look good to me, approved on my end!

**Comment by chrfox** [comment]
publish this on 10/20

**Pull request merged**